### PR TITLE
Proxy all graphql requests through server

### PR DESCRIPTION
**Pull request by camray**

### CODE REVIEW

**Comment by camray** [comment]
#41

File: package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
     "docker:push": "docker push \"altangent/cryptolist:$(cat package.json | grep version | head -1 | awk -F: '{ print $2 }' | sed 's/[\" ,]//g')\""
   },
   "dependencies": {
+    "@altangent/config-manager": "^1.0.2",
```
Comment (bmancini55) [inline package.json]: Will need to update to version `1.0.3`

File: src/server/api/proxy.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+app.post('/proxy/graphql', (req, res, next) => proxyGraphql(req, res).catch(next));
+
+module.exports = app;
+
+///////////////
+
+async function proxyGraphql(req, res) {
+  let { query, variables } = req.body;
+  let config = configMgr.load();
```
Comment (bmancini55) [inline src/server/api/proxy.js]: Should do this on app startup and cache the value.  I usually create a local config.js file that performs the load and exports config as a singleton.

File: package.json (hunk 1 above)
Comment (bmancini55) [inline package.json]: Will need to update to version 1.0.3

File: src/server/api/proxy.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+const express = require('express');
+const configMgr = require('@altangent/config-manager');
+const { GraphQLClient } = require('graphql-request');
+const app = express();
+
+app.post('/proxy/graphql', (req, res, next) => proxyGraphql(req, res).catch(next));
```
Comment (bmancini55) [inline src/server/api/proxy.js]: Minor comment, but I would change this to `/api/graphql` so that it's not immediately obvious that it's a proxied call.

**Pull request merged**